cmd/influx/inspect.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/influxdata/influxdb"
+	"github.com/influxdata/influxdb/internal/fs"
+	"github.com/influxdata/influxdb/tsdb/tsm1"
+	"github.com/spf13/cobra"
+)
+
+// InspectReportTSMFlags defines the `report-tsm` Command.
+type InspectReportTSMFlags struct {
+	pattern  string
+	exact    bool
+	detailed bool
+
+	orgID, bucketID string
+	dataDir         string
+}
+
+var inspectReportTSMFlags InspectReportTSMFlags
+
+func initInspectReportTSMCommand() *cobra.Command {
+	inspectReportTSMCommand := &cobra.Command{
```

Comment:
I've been (slowly) moving over to use https://godoc.org/github.com/influxdata/influxdb/kit/cli.

Here is my example from the godoc: https://godoc.org/github.com/influxdata/influxdb/kit/cli#example-NewCommand

In that library I've added most of the typical viper boilerplate.

Here is a small example from telemetryd:

https://github.com/influxdata/influxdb/blob/master/cmd/telemetryd/main.go#L22-L34

Maybe try it out?